### ISSUE TITLE
Differentiate Between Breaking, Placing, Eating

### ISSUE THREAD

**Issue by viiizee** [Floris0106/Rereskillable-Rereforked#6]
Being able to differentiate between whether I'm trying to eat a carrot or place a carrot would be very nice.  Adding a use case within the quotations that would allow for this would likely be the easiest way in my mind. For example, "minecraft:carrot minecraft:farmland farming:5" would make it so ONLY placing the carrot on a tiled tile is disabled and not eating or general right clicking with the carrot.

**Comment by Floris0106** [comment]
I agree that would be a good feature, but I don't know how I'd go about implementing that. The way you suggested would be really complicated, so I'll try to come up with a better way to approach it. Until then, I don't know whether I want to spend a lot of time on this feature. Anyone is welcome to submit a pull request however!
